tests/tracer/test_logging.py

```diff
@@ -0,0 +1,202 @@
+import os
```

Comment:
The general approach to testing we need is that provided the given inputs/user settings that we properly configure the parent "ddtrace" logger as expected.

We should test that the logger gets configured via `ddtrace-run` or when manually importing `ddtrace`.

When testing with `ddtrace-run`, do not import `ddtrace` so we can validate that the logger gets setup ok on it's own from `ddtrace-run`.

We do not need to assert that other components in the ddtrace package (span, writer, tracer, etc) are logging properly those tests should exist in those component's tests.

We do not need to assert that Python standard library classes/functions are working as expected (e.g. does `RotatingFileHandler` actually rotate the logs?), but we do want to be sure that our logger is setup properly and works as expected (does it actually write to a file?)